CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# History
+
+## 1.1 (2017/03/19)
+
+- Added message bus tags
+- Moved semantic descriptions to `semantic_conventions.md` and `semantic_conventions.yaml`
+- Added OpenTracing governance information to `project_organization.md`
```

Comment:
Maybe within 1.1, release `OTIAB members`, as part of OpenTracing governance. As you mentioned before, `The initial list of OTIAB members will be announced in March 2017.`